Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -566,6 +566,14 @@ theorem measure_mono_null_ae (H : s ≤ᵐ[μ] t) (ht : μ t = 0) : μ s = 0 :=
   nonpos_iff_eq_zero.1 <| ht ▸ H.measure_le
 #align measure_theory.measure_mono_null_ae MeasureTheory.measure_mono_null_ae
 
+/-- Two functions are almost everywhere equal on their entire domain iff they are
+a.e equal on every subsets of their domain. -/
+theorem fun_ae_imp_set_ae {f g : α → β} :
```

Comment:
How do you plan to use this lemma? It's true for any filter, not only `Measure.ae _`, so please prove it for `Filter.EventuallyEq` (or even `Filter.Eventually`).

Reply:
Thanks again for this comment. I find your idea very relevant. I did a general proof for `Filter.Eventually` and specialization for `EventuallyEq` and `EventuallyLE`. However, Don't you think I should do another PR for these results?